domain/src/main/java/no/difi/meldingsutveksling/domain/NextBestAttachement.java

```diff
@@ -0,0 +1,27 @@
+package no.difi.meldingsutveksling.domain;
+
+public class NextBestAttachement implements ByteArrayFile {
+	private ByteArray content;
+	private String filename;
+
+	public NextBestAttachement(ByteArray content, String filename) {
+		this.content = content;
+		this.filename = filename;
+	}
+
+	public NextBestAttachement(byte[] content, String filename) {
+		this(new ByteArray(content), filename);
+	}
+
+	public String getFileName() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1161)